### ISSUE TITLE
maskgit_vqgan‘s vq_loss is incorrect

### ISSUE THREAD

**Issue by yfChang-cv** [bytedance/1d-tokenizer#95]
In the code, the vq_loss is incorrect, and the commitment_cost is placed in the wrong position.

https://github.com/bytedance/1d-tokenizer/blob/bdec006fb7226b309aaf3511956d0aafac30125a/modeling/modules/maskgit_vqgan.py#L296-L299

After correction:
In the code, the vq_loss is fixed, and the commitment_cost is placed in the correct position.

 ```
if return_loss:
    loss = self.commitment_cost  * torch.mean((z_q.detach() - hidden_states) ** 2) +  torch.mean(
        (z_q - hidden_states.detach()) ** 2
    )
```

**Comment by cornettoyu** [comment]
Thanks for pointing out :) This file was borrowed from [OpenMuse](https://github.com/huggingface/open-muse) to load the MaskGIT-VQ pretrained weights. We do not use the loss parts in the file.
